docs/rules/no-duplicate-attr-inheritance.md

```diff
@@ -41,13 +42,41 @@ export default {
   /* ✗ BAD */
   // inheritAttrs: true (default)
 }
+</script>
+```
+
+</eslint-code-block>
+
+### `"checkMultiRootNodes": true`
+
+<eslint-code-block :rules="{'vue/no-duplicate-attr-inheritance': ['error', { checkMultiRootNodes: true }]}">
+
+```vue
+<template>
+  <div v-bind="$attrs" />
+  <div />
+</template>
+<script>
+export default {
+  /* ✗ BAD */
+  // inheritAttrs: true (default)
+}
+</script>
 ```
 
 </eslint-code-block>
 
 ## :wrench: Options
 
-Nothing.
+```json
+{
+  "vue/no-duplicate-attr-inheritance": ["error", {
+    "checkMultiRootNodes": false,
+  }]
+}
+```
+
+- `"checkMultiRootNodes"` ... If `true`, check and warn when there are multiple root nodes. (Default: `false`)
```

Comment:
Can you please add an explanation why the default is `false` (and that is fine), and why someone might still want to enable the option?

Reply:
Add more information and link to the Vue documentation for further explanation.